cdi/src/main/java/io/smallrye/config/inject/ConfigProducerUtil.java

```diff
@@ -128,20 +142,20 @@ static String getRawValue(String name, Config config) {
     }
 
     private static String resolveValue(String name, String defaultValue, Config config) {
```

Comment:
This seems to be used only by the indexed case and called always with null as a default. We should inline any code and remove the method.